src/app/components/drawer/components/drawer-header.tsx

```diff
@@ -29,22 +28,15 @@ export function DrawerHeader({
   const [isHovered, bind] = useHover();
 
   return (
-    <Flex
-      pb="base"
-      justifyContent="space-between"
-      alignItems="center"
-      pt="loose"
-      px="loose"
-      {...bind}
-    >
+    <Flex justifyContent="space-between" alignItems="center" py="space.04" px="space.04" {...bind}>
```

Comment:
It looks like it was `pt` here, but now `py`? If that is the case, can it just be `p="space.04"` bc same?

Reply:
Thanks, I'll update